### PR TITLE
[LRA-15] Update product form

### PR DESCRIPTION
**Pull request by Shalin21**
feat(Update product form): Create update
feat(Update product form): Create notification component
and create product form component
refactor(Update product form): Refactor of current form create form
refactor(Update product form): Refactor of current styled components

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import './App.css';
 import { ProductDetails } from './Components';
 import {homePage, loginPage, signUpPage} from "./Pages/";
 import { createProductPage} from './Pages/CreateProductPage/createProductPage';
+import { updateProductPage} from './Pages/UpdateProductPage/updateProductPage';
```
Comment (mkozachok) [inline src/App.tsx]: please export `createProductPage `and `createProductPage`  from ``./Pages/index.ts

File: src/Components/CreateAntFields/CreateAntFields.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { DatePicker, Form, Input, Select, InputNumber } from "antd";
+import { FieldContainer } from './styled';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+const { TextArea } = Input;
+
+const CreateAntField = AntComponent => ({
```
Comment (mkozachok) [inline src/Components/CreateAntFields/CreateAntFields.js]: miss types

Comment (mkozachok) [inline src/Components/CreateAntFields/CreateAntFields.js]: This it is not component, but function that return component.
Name should start from lowercase.

File: src/Components/CreateAntFields/CreateAntFields.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        >
+          {selectOptions &&
+            selectOptions.map(name => <Option key={name}>{name}</Option>)}
+        </AntComponent>
+      </FormItem>
+    </FieldContainer>
+  );
+};
+
+export const AntSelect = CreateAntField(Select);
```
Comment (mkozachok) [inline src/Components/CreateAntFields/CreateAntFields.js]: 1. Create folder `components/formControls`
2. Put this code to `components/formControls/creators/createAntField.ts`
3. Export all styled fields from `index.ts`

File: src/Components/CreateAntFields/CreateAntUploader.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Form, Upload, Icon, Button, message } from "antd";
+import { FieldContainer } from './styled';
+
+const FormItem = Form.Item;
+
+const CreateAntUploader = () => ({
```
Comment (mkozachok) [inline src/Components/CreateAntFields/CreateAntUploader.js]: Put this code to `components/formControls/creators/createUploader.ts`

File: src/Components/Forms/ProductForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+import { SubmitContainer } from "./styled";
+import { Button } from "antd";
+
+interface IFormikProps {
+  isSubmitting: boolean;
+  isValid: boolean;
+}
+
+export default ({ isSubmitting, isValid }: IFormikProps) => {
+  return (
```
Comment (mkozachok) [inline src/Components/Forms/ProductForm/index.tsx]: return is not required () => value

File: src/Components/Forms/ProductForm/index.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+} from "../../../Components/CreateAntFields/CreateAntFields";
+import { SubmitContainer } from "./styled";
+import { Button } from "antd";
+
+interface IFormikProps {
+  isSubmitting: boolean;
+  isValid: boolean;
+}
+
+export default ({ isSubmitting, isValid }: IFormikProps) => {
```
Comment (mkozachok) [inline src/Components/Forms/ProductForm/index.tsx]: replace default with
export const ProductForm

File: src/Pages/UpdateProductPage/updateProductPage.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { UpdateProductForm } from "../../Components/Forms/UpdateProductForm/UpdateProductForm";
```
Comment (mkozachok) [inline src/Pages/UpdateProductPage/updateProductPage.tsx]: please export from '../../Components/Forms/index.ts'

File: src/Components/CreateAntFields/CreateAntFields.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
```
Comment (mkozachok) [inline src/Components/CreateAntFields/CreateAntFields.js]: Please rename folders
'Components' => '**c**omponents' 
'Pages' => '**p**ages' 
'Layouts' => '**l**ayouts' 
'Forms' => '**f**orms'

**Review by mkozachok — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Shalin21) [reply src/App.tsx]: fixed

File: src/Components/CreateAntFields/CreateAntFields.js (hunk 2 above)
Comment (Shalin21) [reply src/Components/CreateAntFields/CreateAntFields.js]: fixed

Comment (Shalin21) [reply src/Components/CreateAntFields/CreateAntFields.js]: fixed

File: src/Components/CreateAntFields/CreateAntFields.js (hunk 3 above)
Comment (Shalin21) [reply src/Components/CreateAntFields/CreateAntFields.js]: Done

File: src/Components/CreateAntFields/CreateAntUploader.js (hunk 4 above)
Comment (Shalin21) [reply src/Components/CreateAntFields/CreateAntUploader.js]: Done

File: src/Components/Forms/ProductForm/index.tsx (hunk 5 above)
Comment (Shalin21) [reply src/Components/Forms/ProductForm/index.tsx]: fixed

File: src/Components/Forms/ProductForm/index.tsx (hunk 6 above)
Comment (Shalin21) [reply src/Components/Forms/ProductForm/index.tsx]: fixed

File: src/Pages/UpdateProductPage/updateProductPage.tsx (hunk 7 above)
Comment (Shalin21) [reply src/Pages/UpdateProductPage/updateProductPage.tsx]: fixed

File: src/Components/CreateAntFields/CreateAntFields.js (hunk 8 above)
Comment (Shalin21) [reply src/Components/CreateAntFields/CreateAntFields.js]: Done

**Review by mkozachok — APPROVED** [review]

**Pull request merged**